tomcat/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - tomcat
 
+# 1.1.0 / Unreleased
+
+* [Update] Update auto_conf template.
```

Comment:
let's remove the entry from the changelog and put this line in the PR title instead